torchvision/transforms/transforms.py

```diff
@@ -199,21 +199,21 @@ def forward(self, image):
 
 
 class ToPILImage:
-    """Convert a tensor or an ndarray to PIL Image - this does not scale values.
+    """Converts a tensor or an ndarray to PIL Image
 
     This transform does not support torchscript.
 
     Converts a torch.*Tensor of shape C x H x W or a numpy ndarray of shape
-    H x W x C to a PIL Image while preserving the value range.
+    H x W x C to a PIL Image while adjusting the value range depending on the ``mode``.
```

Comment:
@NicolasHug opinion on whether we even want to mention that? 

```suggestion
    H x W x C to a PIL Image.
```

Reply:
Sure, will make the required changes.

Reply:
@NicolasHug All the modes are not mentioned explicitly in the docs. [PIL.Image mode](https://pillow.readthedocs.io/en/latest/handbook/concepts.html#concept-modes) is referred to that in the docs. So won't it be better if we just write this?
```
Converts a torch.*Tensor of shape C x H x W or a numpy ndarray of shape
H x W x C to a PIL Image while keeping the value range consistent with PIL.Image mode.
```

Reply:
Done @NicolasHug